Vagrantfile

```diff
@@ -48,7 +50,7 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |c|
     end
 
     if $mount_users_dir
-      config.vm.synced_folder $home, $home, id: "core", :nfs => true, :mount_options => ['nolock,vers=3,udp']
+      config.vm.synced_folder $home, $home, type: "nfs"
```

Comment:
What I have in mind is that a vagrant image comes from wherever and it might be insecure (maybe intentionally maybe not). So allowing the VM to access you home dir it might not be safe. Also Vagrant VMs might not be accessible from the internet directly but they can access the internet by themselves. I might getting too paranoid about this... (personally I wouldn't even use NFS, I would sync the files only one way from my Mac -> VM through rsync).